content/blog/credits.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Credits for finding security vulnerabilities
+author: Arnout Engelen
+date: 2024-02-22
+description: This post describes when and how we give credit to people who report security issues.
+---
+
+We highly appreciate everyone who responsibly [reports security issues](https://www.apache.org/security/)
+to us: a diverse mix of community members, independent security researchers,
+software auditing firms, etc.
+
+When we release a fix for a vulnerability in one of our projects, we
+publish a security advisory crediting the reporter, and distribute it though
+various Apache [mailinglists](https://www.apache.org/foundation/mailinglists.html), [oss-security](https://www.openwall.com/lists/oss-security/) and the
+[CVE](https://www.cve.org/) programme: this way, we proactively signal the
+urgency to upgrade to our users.
+
+The Apache Software Foundation, being a volunteer organization, does not have
+a bug bounty program. However, some Apache projects are covered under 3rd-party
+bounty programs such as the
+[HackerOne Internet Bug Bounty](https://hackerone.com/ibb).
+
+Notifications about possible security issues in the infrastructure that makes
+development possible, such as websites, CI systems and other supporting
+software, are also very welcome.
+
+There are some classes of common reports that we consider invalid up-front:
+
+* We already know our mailservers do not use DKIM/DMARC. We plan to support this in the future, but this is nontrivial given our strong reliance on mailinglists.
+* As an open source organization with transparency at our core, read access to directory listings, source code repositories and build servers is intentionally public.
```

Comment:
* bulk reports of dependencies coming from security scanners containing no reproducible scenarios how they affect our projects are considered as invalid. Reporters are kindly requested to report individual issues including reproducible scenarios affecting the Apache projects.

Reply:
great idea, split it into two parts (dependency scanner and other code scanner results).